src/components/core/PivotTable.tsx

```diff
@@ -401,12 +409,14 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
         updateTotals: noop,
         onDataTooLarge: noop,
         pageSize: 100,
-        config: {}
+        config: {},
+        groupRows: false
```

Comment:
Do we need default value here? Maybe default value in parent component is enough.

Reply:
we have two PivotTable components:
- Buckets PivotTable -> this is public interface for UI.SDK
- Core PivotTable -> this is used directly by our AD/KD 

I would say it's should be tested on both places as we are using directly this core component :)

Reply:
Ah, okay